### PR TITLE
Gradle Build

### PR DESCRIPTION
**Pull request by lordqwerty**
Builds with gradle. Tests are not running via gradle but looking at `build.xml` tests were not run at build time. 

The gradle build is very basic and needs deployment details added to it via the maven plugin. Will `build.gradle` evolve over time by the community... hopefully.

### CODE REVIEW

**Comment by cblichmann** [comment]
Uh ok, I was working on getting BinNavi to build with Maven instead. Let me check out your Gradle build. OTOH, we probably don't want to maintain two build systems. Or maybe we do, if it's not much work :)

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
-script:
-
-  - mvn dependency:copy-dependencies
-  - ant -f src/main/java/com/google/security/zynamics/build.xml build-binnavi-fat-jar
+language: java
```
Comment (cblichmann) [inline .travis.yml]: Please revert this change, I'd prefer if use the same build commands in the Travis CI build as in the build instructions.

**Comment by cblichmann** [comment]
So, I'll accept this pull request if we keep the original default in the Travis file. We also should make it clear in the README.md/the Gradle file itself that the Gradle build is not the default and that it might break every now and then (as I have no experience with Gradle and Ant/Maven works well enough for now).

**Comment by lordqwerty** [comment]
No problem will sort out now.

**Pull request merged**